src/util/glTF/glTF2Loader.js

```diff
@@ -95,11 +204,18 @@ x3dom.glTF2Loader.prototype._generateX3DNode = function(node)
         } 
     }
 
-    if ( node.name != undefined )
+    if ( node.name === undefined )
     {
+        node.name = this._nodeNamePrefix + index;
+    }
+
+
```

Comment:
Some more commented our code?

Reply:
Yeah, the webVR branch is work in progress.